### PR TITLE
Update ExternalDNS to v0.5.4

### PR DESCRIPTION
**Pull request by linki**
Now that the cache has been fixed and is disabled by default, let's enable and use it 😃

### CODE REVIEW

**Comment by njuettner** [comment]
👍

File: cluster/manifests/external-dns/deployment.yaml [hunk 1]
```diff
@@ -16,20 +16,21 @@ spec:
...
       - name: external-dns
-        image: registry.opensource.zalan.do/teapot/external-dns:v0.5.3
+        image: registry.opensource.zalan.do/teapot/external-dns:v0.5.4
         args:
         - --source=service
         - --source=ingress
         - --provider=aws
         - --registry=txt
         - --txt-owner-id={{ .Region }}:{{ .LocalID }}
+        - --txt-cache-interval=1h
```
Comment (mikkeloscar) [inline cluster/manifests/external-dns/deployment.yaml]: I suggest we fix the problem mentioned here: https://github.com/kubernetes-incubator/external-dns/pull/612#issuecomment-399253828 before we enable caching in our setup.

**Comment by mikkeloscar** [comment]
:+1:

**Comment by linki** [comment]
👍

**Pull request merged**